File: PYTHON/con3.py
"""
  simple Hello world program
  By Itfyme.com
"""

def operation():
    print("enter score")
    score=int(input())
    if score<=100 and score>=90:
        print("A")
    elif score<=90 and score>=80:
        print("B")
    elif score<=80 and score>=70:
        print("C")  
    elif score<=70 and score>=50:
        print("D")
    elif score<=50 and score>=30:
        print("E")  
    elif score<=30 and score>=0:
        print("F")
  
  
operation()

File: C/con3.c
#include <stdio.h>
int main(int argc, char *argv[]) {
  int score;
  printf("enter score\n");
  scanf("%d",&score);
  if(score<=100 && score>=90){
      printf("A\n");
  }
  else if(score<=90 && score>=80){
      printf("B\n");
  }
    else if(score<=80 && score>=70){
      printf("C\n");
  }
    else if(score<=70 && score>=50){
      printf("D\n");
  }
    else if(score<=50 && score>=30){
      printf("E\n");
  }
    else if(score<=30 && score>=0){
      printf("F\n");
  }
}